arlo-client/src/components/DataEntry/index.tsx

```diff
@@ -59,18 +59,19 @@ const loadBallots = async (
   return ballots
 }
 
-const saveMembers = async (
+const putMembers = async (
   electionId: string,
   jurisdictionId: string,
+  roundId: string,
   auditBoardId: string,
   members: IAuditBoardMember[]
 ) => {
   try {
     await api(
-      `/election/${electionId}/jurisdiction/${jurisdictionId}/audit-board/${auditBoardId}`,
+      `/election/${electionId}/jurisdiction/${jurisdictionId}/round/${roundId}/audit-board/${auditBoardId}`,
       {
-        method: 'POST',
-        body: JSON.stringify({ members }),
+        method: 'PUT',
+        body: JSON.stringify(members),
```

Comment:
Did I miss that we're moving to JSON arrays? Not my cup of tea, though not gonna cry about it.

Reply:
Funny you mention it. I'm fairly convinced that the envelope pattern is unnecessary, and am starting to not use it.

However, in this case I think you're right that I was overzealous. Really this is only updating part of the audit board object, so the body should be `{members: [...]}`. I think technically the method should be `PATCH` also. Do you have thoughts about that?